### DISCUSSION TITLE
Nova support

### DISCUSSION THREAD

**Discussion by mrdeniska1234** [DerFrZocker/Custom-Ore-Generator#31] [Q&A] [answered]
How add ores from Nova to this ore generator? All my attempts are useless; a mushroom block without texture is generated, instead of a block with texture. Once I managed to generate blocks with a custom texture, but when the block was destroyed it behaved like a mushroom block.
 As far as I understand, there are not enough NBT Tags, but when adding them to
```
custom-data: 
NBT_TAG: '{nova: {id: "machines:star_shards_ore", subId: 0}}'
```
this didn't give any results

**Comment by DerFrZocker** [comment, marked as answer]
After halving a very quick lock at the plugin, it seems that nova saves some additional data, which this plugin cannot do, nova support would probably require an extra hook.

**Reply by mrdeniska1234** [reply]
Thanks for the answer. Please tell me if this will be implemented in the future?
